deployment/docker/docker-compose.yml

```diff
@@ -68,6 +68,7 @@ services:
       - ./alerts-health.yml:/etc/alerts/alerts-health.yml
       - ./alerts-vmagent.yml:/etc/alerts/alerts-vmagent.yml
       - ./alerts-vmalert.yml:/etc/alerts/alerts-vmalert.yml
+      - ./slo-rules.yml:/etc/alerts/slo-rules.yml
```

Comment:
what is this new file?

Reply:
hm, I made a test and probably added it accidentally